P2P Integration test randomly fail

### Expected behavior
Integration test should be deterministic and not fail

### Actual behavior
Jenkins still fails at below
```
Integration tests for P2P library
Partially connected network which becomes fully connected: All nodes launch at the same time. The seedPeers list of each node contains the next node in the sequence. Discovery interval runs multiple times.
Peer discovery
should discover all peers in the network after a few cycles of discovery:
AssertionError: expected [ Array(9) ] to deeply equal [ Array(10) ]

at p2pNodeList.forEach.p2p (test/integration/p2p.ts:198:30)
at Array.forEach (<anonymous>)
at Context.it (test/integration/p2p.ts:192:17)
```

--- a/packages/lisk-p2p/test/integration/p2p.ts
+++ b/packages/lisk-p2p/test/integration/p2p.ts
@@ -330,6 +330,7 @@ describe('Integration tests for P2P library', () => {
 				return new P2P({
 					blacklistedPeers: [],
 					connectTimeout: 5000,
+					ackTimeout: 5000,
 					seedPeers,
 					wsEngine: 'ws',
 					nodeInfo: {
@@ -682,7 +683,6 @@ describe('Integration tests for P2P library', () => {
 	});
 
 	describe('Connected network: User custom selection algorithm is passed to each node', () => {
-		const DISCOVERY_INTERVAL = 200;
 		// Custom selection function that finds peers having common values for modules field for example.
 		const peerSelectionForSendRequest: P2PPeerSelectionForSendRequest = (
 			peersList: ReadonlyArray<P2PDiscoveredPeerInfo>,
@@ -744,9 +744,9 @@ describe('Integration tests for P2P library', () => {
 				return new P2P({
 					blacklistedPeers: [],
 					connectTimeout: 5000,
+					ackTimeout: 5000,
 					peerSelectionForSendRequest,
 					peerSelectionForConnection,
-					discoveryInterval: DISCOVERY_INTERVAL,
 					seedPeers,
 					wsEngine: 'ws',
 					nodeInfo: {
@@ -903,8 +903,8 @@ describe('Integration tests for P2P library', () => {
 					seedPeers,
 					wsEngine: 'ws',
 					// A short connectTimeout and ackTimeout will make the node to give up on discovery quicker for our test.
-					connectTimeout: 100,
-					ackTimeout: 100,
+					connectTimeout: 1000,
+					ackTimeout: 1000,
 					// Set a different discoveryInterval for each node; that way they don't keep trying to discover each other at the same time.
 					discoveryInterval: DISCOVERY_INTERVAL + index * 11,
 					nodeInfo: {

--- a/packages/lisk-p2p/test/integration/p2p.ts
+++ b/packages/lisk-p2p/test/integration/p2p.ts
@@ -94,8 +94,8 @@ describe('Integration tests for P2P library', () => {
 					seedPeers,
 					wsEngine: 'ws',
 					// A short connectTimeout and ackTimeout will make the node to give up on discovery quicker for our test.
-					connectTimeout: 100,
-					ackTimeout: 100,
+					connectTimeout: 1000,
+					ackTimeout: 1000,
 					// Set a different discoveryInterval for each node; that way they don't keep trying to discover each other at the same time.
 					discoveryInterval: DISCOVERY_INTERVAL + index * 11,
 					nodeInfo: {

